neuraxle/steps/misc.py

```diff
@@ -161,7 +160,7 @@ def inverse_transform(self, processed_outputs):
         return processed_outputs
 
 
-class CallbackWrapper(ForceMustHandleMixin, MetaStepMixin, BaseStep):
+class CallbackWrapper(HandlerMixin, MetaStepMixin, BaseStep):
```

Comment:
```
class OnlyHandleMixin

class RouteVanillaToHandlesMixin

class ForceHandleMixin(RouteVanillaToHandlesMixin, OnlyHandleMixin):
    pass  # Or AlwaysHandleMixin
```

NOTE: in every comment above in the PR review when I was referring to ForceHandleMixin, I was talking of the OnlyHandleMixin. Here the ForceHandleMixin is a new one.

Reply:
So with the new `ForceHandleMixin`, you might change some of the previous OnlyHandleMixin to some ForceHandleMixin.